core/src/main/java/org/apache/cxf/io/DelayedCachedOutputStreamCleaner.java

```diff
@@ -226,11 +235,19 @@ public void initComplete() {
     
     @Override
     public void postShutdown() {
+        // If cleanup on shutdown is asked, force cleaning all cached output streams
+        if (cleanupOnShutdown) {
+            forceClean();
```

Comment:
@coheigea minor improvement to forceably cleanup all cached output streams upon `Bus` shutdown